unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -270,9 +311,24 @@ public async UniTask<string[]> GetFriendsAsync(int limit, int skip, Cancellation
             return payload.friends;
         }
 
-        public async UniTask<string[]> GetFriendsAsync(string usernameOrId, int limit, CancellationToken cancellationToken = default)
+        public async UniTask<string[]> GetFriendsAsync(string userNameOrId, int limit, CancellationToken cancellationToken = default)
         {
-            var payload = await apiBridge.GetFriendsAsync(usernameOrId, limit, cancellationToken);
+            if (useSocialApiBridge)
+            {
+                var friendsToReturn = new List<string>();
+
+                foreach (var friend in friendsSortedByName.Values)
+                {
+                    if (friend.userName.IndexOf(userNameOrId, StringComparison.OrdinalIgnoreCase) >= 0 ||
```

Comment:
I wonder why you use `IndexOf` instead of equality comparison?

Reply:
Because this allows for finding a string in the middle of the userName not just equal or beggining